scio-jdbc/src/main/scala/com/spotify/scio/jdbc/package.scala

```diff
@@ -0,0 +1,153 @@
+package com.spotify.scio
+
+import java.sql.{Driver, PreparedStatement, ResultSet}
+
+import com.spotify.scio.Implicits._
+import com.spotify.scio.io.Tap
+import com.spotify.scio.testing.TestIO
+import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.io.jdbc.JdbcIO
+import org.apache.beam.sdk.io.jdbc.JdbcIO._
+
+import scala.concurrent.Future
+import scala.reflect.ClassTag
+
+package object jdbc {
```

Comment:
Add some doc for the package object like the ones in `com.spotify.scio.bigtable`. Like importing all and stuff.